### ISSUE TITLE
Sidekiq params

### ISSUE THREAD

**Issue by lunaru** [pat/ts-sidekiq-delta#11]
@pat I've put up a PR to address some errors related to `unique` vs `lock`: https://github.com/pat/ts-sidekiq-delta/pull/10

**Comment by lunaru** [comment]
Here's a dump of the error, which I've partially redacted:

```
[af222b9c-150f-4178-b40d-05cbadcddaca] undefined method `to_sym' for true:TrueClass
Did you mean?  to_s
[af222b9c-150f-4178-b40d-05cbadcddaca] /home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-unique-jobs-6.0.13/lib/sidekiq_unique_jobs/options_with_fallback.rb:52:in `lock_class'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-unique-jobs-6.0.13/lib/sidekiq_unique_jobs/options_with_fallback.rb:45:in `lock'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-unique-jobs-6.0.13/lib/sidekiq_unique_jobs/client/middleware.rb:43:in `block in locked?'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-5.2.7/lib/sidekiq/logging.rb:48:in `with_context'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-unique-jobs-6.0.13/lib/sidekiq_unique_jobs/sidekiq_unique_jobs.rb:41:in `with_context'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-unique-jobs-6.0.13/lib/sidekiq_unique_jobs/client/middleware.rb:42:in `locked?'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-unique-jobs-6.0.13/lib/sidekiq_unique_jobs/client/middleware.rb:37:in `success?'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-unique-jobs-6.0.13/lib/sidekiq_unique_jobs/client/middleware.rb:27:in `call'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-5.2.7/lib/sidekiq/middleware/chain.rb:130:in `block in invoke'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-5.2.7/lib/sidekiq/middleware/chain.rb:133:in `invoke'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-5.2.7/lib/sidekiq/client.rb:212:in `process_single'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-5.2.7/lib/sidekiq/client.rb:71:in `push'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-5.2.7/lib/sidekiq/worker.rb:149:in `client_push'
/home/vagrant/bundle/ruby/2.4.0/gems/sidekiq-5.2.7/lib/sidekiq/worker.rb:93:in `perform_async'
/home/vagrant/bundle/ruby/2.4.0/gems/ts-sidekiq-delta-0.2.0/lib/thinking_sphinx/deltas/sidekiq_delta.rb:59:in `index'
```

**Comment by pat** [comment]
Presuming this can be closed now, but please do confirm v0.3.0 does fix things for you first.

**Comment by lunaru** [comment]
Yup all good!
